packages/charts/src/charts/bar/bar-chart.component.tsx

```diff
@@ -207,17 +210,16 @@ export const BarChart: FC<Props> = ({
               hasSpacing={false}
             >
               {tooltip.selectors && (
-                <BarTooltipContent
+                <BarChartTooltip
                   data={localizedData}
                   keys={keys}
                   disabledKeys={disabledKeys}
-                  stackMode={stackMode}
-                  groupMode={groupMode}
+                  isPercentage={isPercentage}
```

Comment:
What do You think about move it as `inline` instead of const (especially it's used only in one place).
```
<BarChartTooltip
   isPercentage={stackMode === 'percent' && groupMode === 'stacked'}
  ...
/>
```